heartbeat/monitors/task.go

```diff
@@ -20,6 +20,7 @@ package monitors
 import (
 	"context"
 	"fmt"
+	"github.com/elastic/beats/v7/x-pack/functionbeat/function/core"
```

Comment:
What I would like to see is moving `core.SyncClient` to libbeat, so everyone can use it. Also, it's not a good practice to use code from a different Beat.

Reply:
Ah, I missed that the import comes straight from functionbeat, we'll try and make that move in this PR, I agree it's ugly